src/4626/CompoundERC4626.sol

```diff
@@ -187,7 +223,7 @@ contract CompoundERC4626 is ERC4626 {
             uint256 currentExchangeRate = mToken.viewExchangeRate();
             uint256 _totalSupply = MToken(address(mToken)).totalSupply();
             uint256 totalSupplies = (_totalSupply * currentExchangeRate) / 1e18; /// exchange rate is scaled up by 1e18, so needs to be divided off to get accurate total supply
-    
```

Comment:
can't seem to get rid of this edit, but the commented code below i did not add or remove. seems relevant context for the comments that follow.